src/test/kotlin/uk/gov/dluhc/printapi/rest/aed/GetAnonymousElectorDocumentsByApplicationIdIntegrationTest.kt

```diff
@@ -174,4 +174,30 @@ internal class GetAnonymousElectorDocumentsByApplicationIdIntegrationTest : Inte
             .usingRecursiveComparison()
             .isEqualTo(listOf(expectedFirstRecord, expectedSecondRecord))
     }
+
+    @Test
+    fun `should return empty list given no AEDs exist for application ID`() {
+        // Given
+        val eroResponse = buildElectoralRegistrationOfficeResponse(
+            id = ERO_ID,
+            localAuthorities = listOf(buildLocalAuthorityResponse(gssCode = GSS_CODE), buildLocalAuthorityResponse())
+        )
+        wireMockService.stubCognitoJwtIssuerResponse()
+        wireMockService.stubEroManagementGetEroByEroId(eroResponse, ERO_ID)
+
+        // When
+        val response = webTestClient.get()
+            .uri(URI_TEMPLATE, ERO_ID, APPLICATION_ID)
+            .bearerToken(getVCAnonymousAdminBearerToken(eroId = ERO_ID))
+            .contentType(APPLICATION_JSON)
+            .exchange()
+            .expectStatus().isOk
+            .returnResult(AnonymousElectorDocumentsResponse::class.java)
+
+        // Then
+        val actual = response.responseBody.blockFirst()
+        assertThat(actual).isNotNull
```

Comment:
Minor, but I dont think you need these 2 nonNull assertions
I think you could get away with just L201 👍